src/invidious/comments.cr

```diff
@@ -554,7 +558,7 @@ def content_to_comment_html(content)
 
         if url.host == "youtu.be"
           url = "/watch?v=#{url.request_target.lstrip('/')}"
-        elsif !url.host || {"m.youtube.com", "www.youtube.com"}.includes? url
+        elsif !url.host || {"m.youtube.com", "www.youtube.com"}.includes? url.host
```

Comment:
I've though about that, and we should use `ends_with` instead. As it's mostly used in description and comments, it's not rare to see youtube URLs with the `www` or `m` subdomain missing.